easier68k/core/opcodes/ori.py

```diff
@@ -0,0 +1,307 @@
+from ...core.enum.ea_mode import EAMode
+from ...core.models.assembly_parameter import AssemblyParameter
+from ...core.enum import ea_mode_bin
+from ...core.enum.ea_mode_bin import parse_ea_from_binary
+from ...simulator.m68k import M68K
+from ...core.util.split_bits import split_bits
+from ...core.opcodes.opcode import Opcode
+from ...core.util import opcode_util
+from ...core.enum.op_size import OpSize
+from ..util.parsing import parse_assembly_parameter
+from ..enum.condition_status_code import ConditionStatusCode
+from ..models.memory_value import MemoryValue
+
+class Ori(Opcode):  # Forward declaration
+    pass
+
+
+class Ori(Opcode):
+    valid_sizes = [OpSize.BYTE, OpSize.WORD, OpSize.LONG]
+
+    def __init__(self, params: list, size: OpSize=OpSize.WORD):
+        assert len(params) == 2
+        assert isinstance(params[0], AssemblyParameter)
+        assert isinstance(params[1], AssemblyParameter)
+
+        # check src param is valid
+        assert params[0].mode == EAMode.IMM
+        self.src = params[0]
+
+        # check the dest param is valid
+        assert params[1].mode != EAMode.ARD or params[1].mode != EAMode.IMM
+        self.dest = params[1]
+
+        assert size in Ori.valid_sizes
+        self.size = size
+
+    def assemble(self) -> bytearray:
+        """
+        Assembles this opcode into hex to be inserted into memory
+        :return: The hex version of this opcode
+        """
+        ret_opcode = 0b00000000 << 8
```

Comment:
I think that just doing
```py
ret_opcode = 0
```
and explaining that the first 8 bits are all zero should be fine. Manual says that this is correct, though.